beets/art.py

```diff
@@ -189,20 +189,26 @@ def check_art_similarity(log, item, imagepath, compare_threshold):
     return True
 
 
-def extract(log, outpath, item):
+def extract(log, item, outpath=None):
+    """Extract art from item into outpath, or into a tmp file if unset.
```

Comment:
Perhaps explain here that `outpath` will get an extension applied automatically.

Reply:
Done.